common/debug.c

```diff
@@ -22,6 +22,9 @@ static char *prefixes[] = {
 #if DEBUG
 	"DEBUG:\t ",
 #endif
+#if TEST
```

Comment:
When you rebase, I'll have removed the `#if` around `DEBUG`. Remove the `#if` here as well.